### PR TITLE
Solaredge documentation for #26282

### PR DESCRIPTION
**Pull request by cereal2nd**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#26282

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/10256"></a>

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_components/solaredge.markdown [hunk 1]
```diff
@@ -20,7 +20,17 @@ The SolarEdge Monitoring API has a daily rate limit of 300 requests. In order to
...
-To use the SolarEdge sensors in your installation, add the following to your configuration.yaml file:
+There are 2 options in configuring the SolarEdge integration:
+
+- Via the Home Assistant user interface where it will let you enter the port string to connect to the Velbus bus.
+- Via the Home Assistant `configuration.yaml` file.
+
+<div class='note'>
+
+If the integration is enabled via the user interface all monitored conditions are enabled by default.
+When configuring via configuration.yaml you can select the monitored conditions to be translated to a sensor.
```
Comment (MartinHjelmare) [inline source/_components/solaredge.markdown]: We removed the monitored conditions.

**Review by MartinHjelmare — CHANGES_REQUESTED** [review]
Please update the page regarding monitored_conditions.

File: source/_components/solaredge.markdown (hunk 1 above)
Comment (MartinHjelmare) [reply source/_components/solaredge.markdown]: This needs to be updated still.

**Review by MartinHjelmare — DISMISSED** [review]

File: source/_components/solaredge.markdown [hunk 2]
```diff
@@ -46,75 +48,19 @@ name:
...
-      - meters
-      - sensors
-      - gateways
-      - batteries
-      - inverters
-      - power_consumption
-      - solar_power
-      - grid_power
-      - storage_power
+solaredge:
```
Comment (klaasnicolaas) [inline source/_components/solaredge.markdown]: I think we can skip this full example

Comment (frenck) [reply source/_components/solaredge.markdown]: Agreed, it doesn't add much now anymore. Removed that one.

**Review by frenck — APPROVED** [review]
Thanks for the PR @Cereal2nd! 👍

**Pull request merged**